relationshipTypes/membership.yml

```diff
@@ -5,26 +5,45 @@ description: A member is part of a group
 roleTypes:
   - '@type': RoleType
     id: roleTypes/member
+    color: "#3F51B5"
     name: member
     pluralName: members
     relationshipType: relationshipTypes/membership
-    label: "People in {target}"
+    linkType: linkTypes/belongs-to
+    queryLabels: "a member in {context}"
   - '@type': RoleType
     id: roleTypes/group
+    color: "#2196F3"
     name: group
     pluralName: groups
     relationshipType: relationshipTypes/membership
-    label: "Groups that include {target}"
+    linkType: linkTypes/includes
+    queryLabels: "a group in {context}"
 links:
   - '@type': LinkType
-    id: linkTypes/is-member-of
+    id: linkTypes/belongs-to
+    inverse: linkTypes/includes
+    color: "#3F51B5"
     relationshipType: relationshipTypes/membership
-    source: roleTypes/member
-    target: roleTypes/group
-    label: "is member of {target}"
+    source: /roleTypes/member
+    target: /roleTypes/group
+    labels: "{source} belongs to {target}"
+      # - "{source} belongs to {target}"
+      # - "{source} is part of {target}"
+    queryLabels: "People who belong to {target},{source}'s groups"
+      # target:
+        # - "People who belong to {target}"
+        # - "People who are part of {target}"
+
   - '@type': LinkType
-    id: linkTypes/has-member
+    id: linkTypes/includes
```

Comment:
i don't have a problem with the verbs on their own, but i feel the verbs for 'membership' and 'set' are describing more or less the same concept, which is confusing.